tests/vcs/git/GitCommitBearTest.py

```diff
@@ -139,6 +139,17 @@ def test_shortlog_checks_shortlog_trailing_period(self):
         self.assertEqual(self.run_uut(shortlog_trailing_period=False), [])
         self.assertEqual(self.run_uut(shortlog_trailing_period=None), [])
 
+    def test_shortlog_checks_imperative(self):
+        self.git_commit("tag: Add shortlog in imperative")
+        self.assertNotIn("Shortlog of HEAD commit isn't imperative mood",
+                         ''.join(self.run_uut()))
+        self.git_commit("Added invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
+        self.git_commit("Adding another invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
```

Comment:
hm false positives are an issue and IMO worse than false negatives, @tushar-rishav can you write some documentation about all this into the doc comment? Then it's visible to the user too.